HIRS_AttestationCAPortal/src/main/webapp/WEB-INF/jsp/rim-details.jsp

```diff
@@ -57,7 +57,14 @@
                                 <c:if test="${not empty initialData.events}">
                                     <c:set var="count" value="1" scope="page"/>
                                     <c:forEach items="${initialData.events}" var="event">
-                                        <tr>
+                                        <c:choose>
+                                            <c:when test="${event.isError()}">
+                                                <tr style="background: tomato">
+                                                </c:when>
+                                                <c:otherwise>
+                                                <tr>
+                                                </c:otherwise>
+                                            </c:choose>
```

Comment:
Nit: whitespace

Reply:
I guess you mean the format of the lines?